### PR TITLE
solaar: 1.0.2 -> 1.0.5

### PR DESCRIPTION
**Pull request by peterhoeg**
###### Motivation for this change

Upstream update.

###### Things done

- [X] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [X] Ensured that relevant documentation is up to date
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by peterhoeg** [comment]
Also throw in a proper release of `ltunify` which now sees nano receivers.

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 122424` at 1ba244d3 run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>3 packages built successfully:</summary>
  <ul>
    <li>logitech-udev-rules</li>
    <li>ltunify</li>
    <li>solaar</li>
  </ul>
</details>
<details>
  <summary>2 suggestions:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/unclear-gpl.md">unclear-gpl</a>

`gpl2` is a deprecated license, please check if project uses `gpl2Plus` or `gpl2Only` and change `meta.license` accordingly.

Near pkgs/os-specific/linux/logitech-udev-rules/default.nix:16:26:
```
   |
16 |     inherit (solaar.meta) homepage license platforms;
   |                          ^
```
</li>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/no-python-tests.md">no-python-tests</a>

Test runner could not discover any test cases: ‘Ran 0 tests in 0.000s’
Near pkgs/applications/misc/solaar/default.nix:40:0:
```
   |
40 |     description = "Linux devices manager for the Logitech Unifying Receiver";
   | ^
```
</li>
  </ul>
</details>

File: pkgs/tools/misc/ltunify/default.nix [hunk 1]
```diff
@@ -1,17 +1,17 @@
 { lib, stdenv, fetchFromGitHub }:
```
Comment (spinus) [inline pkgs/tools/misc/ltunify/default.nix:2]: imho that should be separate PR

Comment (peterhoeg) [reply pkgs/tools/misc/ltunify/default.nix:2]: In principle, yes. These are however 2 tiny but related packages and in case we need to revert anything, it's in a separate commit.

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 122424` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>3 packages built:</summary>
  <ul>
    <li>logitech-udev-rules</li>
    <li>ltunify</li>
    <li>solaar</li>
  </ul>
</details>

File: pkgs/applications/misc/solaar/default.nix [hunk 2]
```diff
@@ -1,40 +1,50 @@
 { fetchFromGitHub, lib, gobject-introspection, gtk3, python3Packages }:
-# Although we copy in the udev rules here, you probably just want to use logitech-udev-rules instead of
-# adding this to services.udev.packages on NixOS
+
+# Although we copy in the udev rules here, you probably just want to use
+# logitech-udev-rules instead of adding this to services.udev.packages on NixOS
```
[the remark this answers is not in the document]
Comment (spinus) [reply pkgs/applications/misc/solaar/default.nix:4]: sorry, that was not my point. My point was that if somebody installs the package and it doesn't work, the description says what steps need to be done to enable udev rules (it can be to enable hardware.logitech, sure)

File: pkgs/tools/misc/ltunify/default.nix (hunk 1 above)
Comment (spinus) [reply pkgs/tools/misc/ltunify/default.nix:2]: if somebody who can merge is happy with that, let's go

File: pkgs/applications/misc/solaar/default.nix (hunk 2 above)
[the remark this answers is not in the document]
Comment (peterhoeg) [reply pkgs/applications/misc/solaar/default.nix:4]: How about now?

[the remark this answers is not in the document]
Comment (spinus) [reply pkgs/applications/misc/solaar/default.nix:4]: looks cool, thank you for taking that into account
